README.md

```diff
@@ -32,13 +32,29 @@ Achieve a position on the leader boards at the end of each week and earn FitCoin
 | stravaBotId        | Name of the Strava bot account                                   |
 | stravaClientId     | Strava OAuth ID of the bot account                               |
 | stravaRefreshToken | Strava OAuth refresh token of the bot account                    |
-| stravaAccessToken  | Strava OAuth access token of the bot account                     |
 | stravaClientSecret | Strava OAuth client secret token of the bot account              |
 | stravaClubs        | Array of Strava clubs to monitor                                 |
 
 An example config upload is provided in [upload-config.ts][upload-config].
 
-Note that the scope of the Strava OAuth token must be set to `read,activity:read,activity:write`. More details on [Strava authentication documentation][strava-oauth] can be found on their website.
+### Strava OAuth Refresh Token
```

Comment:
We might want to add how to add an app to a Strava account: https://developers.strava.com/docs/getting-started/#account